### PR TITLE
Fix #235: Improved the UI if no story is found

### LINKED ISSUE #235: [Feature] Improve design when no story is found
## Is your feature request related to a problem? Please describe.

Currently the design when no story is available looks like this
image

## Describe the solution you'd like

The design should be improved to fit to the UI. Also, it should prompt the user to create a new story in that case.

## Additional context

Link to new design:

https://www.figma.com/file/24a518w9Avz8pCCBKZhUNP/User-Story---No-Stories?node-id=0%3A1

### PR DESCRIPTION
**Pull request by kailash360**
**Issue Number**

fixes #235 

**Describe the changes you've made**

I have modified the UI if no story is found in the story list. I have also added a link to create a new story from the story list.

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

NA

**Additional context (OPTIONAL)**

NA

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

```cmd
npm run test
```

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**

Preview: Deploy preview link here with the appropriate route

image

### CODE REVIEW

**Review by cyntss — CHANGES_REQUESTED** [review]

File: src/assets/scss/components/index.scss [hunk 1]
```diff
@@ -23,3 +23,4 @@
 @import 'search';
 @import 'storyPageTimeline';
 @import 'productList.scss';
+@import './noStory.scss';
```
[the remark this answers is not in the document]
Comment (kailash360) [reply src/assets/scss/components/index.scss]: Alright

File: src/components/NoStory.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+function NoStory() {
+  return (
+    <div className='no-story'>
+      <p>No story found</p>
+      <a href='/newStory' className='link'>
```
[the remark this answers is not in the document]
Comment (kailash360) [reply src/components/NoStory.jsx]: Done

File: src/assets/scss/components/noStory.scss [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.no-story {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 200px;
+  min-width: 100%;
+
+  & p {
```
[the remark this answers is not in the document]
Comment (kailash360) [reply src/assets/scss/components/noStory.scss]: Sure, will keep that in mind

**Comment by cyntss** [comment]
@kailash360 check that there are merge conflicts in the PR after my PR was merged. Let me know when you fix them so I can approve this PR

**Comment by kailash360** [comment]
> @kailash360 check that there are merge conflicts in the PR after my PR was merged. Let me know when you fix them so I can approve this PR

@cyntss 
I've resolved the conflicts and pushed

**Comment by kailash360** [comment]
> @kailash360 check that there are merge conflicts in the PR after my PR was merged. Let me know when you fix them so I can approve this PR

@cyntss 
I've resolved the conflicts and pushed

**Pull request merged**